src/actions/geek-lab/config.js

```diff
@@ -0,0 +1,35 @@
+exports.command = 'config';
+exports.describe = 'show and perform changes on config file';
+
+exports.builder = (yargs) => yargs
+  .example('$0 config', 'shows cli tool config file')
+  .example('$0 config --env prod --key apiUrl --value blah', 'sets apiUrl on env prod to be blah')
+  .option('env', { describe: 'environment', type: 'string' })
+  .option('key', { describe: 'key that will receive the value', type: 'string' })
+  .option('value', { describe: 'value...', type: 'string' });
+
+const {
+  readConfig,
+  getUserDirectory,
+  writeInternalCliFile,
+} = require('../../lib/utils');
+exports.handler = (argv) => {
+  const configFileContent = readConfig();
+  if (argv.env && argv.key && argv.value) {
+    if (!configFileContent[argv.env]) {
+      configFileContent[argv.env] = {};
+    }
+    configFileContent[argv.env][argv.key] = argv.value;
+    writeInternalCliFile(
```

Comment:
leveraging a function previouslycreated i can just say the file name and the content, and everything else is handled